main.py

```diff
@@ -13,25 +13,23 @@
 word_list_df = pd.read_csv(f"{path}{file_name}", sep =';')
 
 # Quick data check. What are we working with.
-# print(nseant_df['Antwoord'][50])
+# print(nseant_df['Antwoord'][50],)
 # print(word_list_df['WoordenLijst'][50])
-# print(type(nseant_df))
 
-# Expand NSE open answers. Add columns for AVG sensitivity and AVG sensitive words.
+# Add columns: 1 for AVG sensitivity and 1 for AVG sensitive words.
 ## AVG sensitivity is either 0 or 1. preset 0 is not sensitive.
 nseant_df['AVG_gevoelig'] = 0
```

Comment:
Zou je deze standaard op 1 willen zetten? Dus: voordat we beginnen gaan we er vanuit dat alles AVG gevoelige informatie bevat, en we gaan bewijzen dat dit voor een aantal gevallen niet zo is.